src/SFA.DAS.Tools.Servicebus.Support.Web/Views/MessageList/Index.cshtml

```diff
@@ -29,32 +23,6 @@
             return res;
         }
 
-        function sendRequest(url, data) {
-            displayBusyIndicator();
-
-            var form = $('#__AjaxAntiForgeryForm');
-            var token = $('input[name="__RequestVerificationToken"]', form).val();
-
-            $.ajax({
-                type: "POST",
-                url: url,
-                dataType: 'json',
-                data: {
-                    '__RequestVerificationToken': token,
-                    "data": JSON.stringify(data)
-                },
-                failure: function (response) {
-                    console.log(response.responseText);
-                },
-                error: function (response) {
-                    console.log(response.responseText);
-                }
-            })
-                .then(function () {
-                    location.reload();
-                });
-        }
-
         $(function () {
             $table.on('check.bs.table check-all.bs.table uncheck.bs.table uncheck-all.bs.table',
```

Comment:
Is this function to maintain selections during pagination? if so you can do away with the entire function ( and the search clear function below ) by adding `data-maintain-meta-data="true"` to your bootstrap table definition. Then wherever you need the Id's:

`var selectedRows = $.map(table.bootstrapTable('getSelections'), function (row) {
                    return row.id
                });`

Will return you an array of the Id columns selected

Reply:
I doesn't look like data-maintain-meta-data works with server side paging